environments/mcdTransformers.toml

```diff
@@ -188,6 +190,14 @@
         migrations = "db/migrations"
         contracts = ["MCD_FLIP_ETH_A", "MCD_FLIP_BAT_A", "MCD_FLIP_SAI", "MCD_FLOP"]
         rank = "0"
+    [exporter.deny]
+        path = "transformers/events/deny/initializer"
+        type = "eth_event"
+        repository = "github.com/makerdao/vdb-mcd-transformers"
+        migrations = "db/migrations"
+        contracts = ["MCD_CAT", "MCD_FLAP", "MCD_FLIP_ETH_A", "MCD_FLIP_BAT_A", "MCD_FLIP_SAI", "MCD_FLOP", "MCD_JUG",
+                     "MCD_POT", "MCD_SPOT", "MCD_VOW", "OSM_ETH", "OSM_BAT"]
```

Comment:
Hmm, maybe I'm looking in the wrong place? Is this the correct OSM implementation?

https://github.com/makerdao/osm/blob/master/src/osm.sol#L39

Reply:
Looking in etherscan, I can see that the contract at the address specified in the changelog doesn't have the `note` modifier, so you're totally right. Wish I could figure out how to get from a release to that version's OSM contract in GitHub though 🤔

Reply:
It looks like the way that contract versions in dss-deploy-scripts was changed as of [2.16](https://github.com/makerdao/dss-deploy-scripts/commit/a4171ffde33a76417bb9c97a2b2c8890aef6cd71). Instead of looking in the contracts directory for, we now can look at the [.dapp.json](https://github.com/makerdao/dss-deploy-scripts/blob/0.2.17/.dapp.json) file which includes the revision for the dss, osm and cdp_manager repos.

Reply:
for osm, here's the revision that was used for 2.17: https://github.com/makerdao/osm/commit/504c47437916e29d918a9d1f40eb1f7595f3e9ce

Reply:
Thanks, that's super helpful!